ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Common/vars/RedHat.yml

```diff
@@ -55,8 +55,21 @@ Additional_Build_Tools_RHEL7:
 Additional_Build_Tools_RHEL7_PPC64LE:
   - libstdc++.so.5
 
-Additional_Build_Tools_RHEL_x86_64:
+C_Library_32bit_RHEL_x86_64:
+  - glibc.i686
   - glibc-devel.i686
+  - libstdc++.i686
+
+C_Library_32bit_RHEL_ppc64:
+  - glibc.ppc
+  - glibc-devel.ppc
+  - libstdc++.ppc
+  - libstdc++-devel.ppc
```

Comment:
`libstdc++-devel.ppc` seems out of place comparing to the others.  Is there any explanation why this would be needed for just Linux PPC BE and not others?

Reply:
Hey Violeta! libstdc++-devel.ppc is on the list when I did the additional tools for ppc64. Do you happen to know why we need that? Thanks! @vsebe

Reply:
Not need it.